### DISCUSSION TITLE
Welcome to FT8CN Discussions!

### DISCUSSION THREAD

**Discussion by N0BOY** [N0BOY/FT8CN#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kj7prs** [comment]
How about a quick little operating manual.  At least what the buttons do?  Love it so far.

**Reply by UT8IIV** [reply]
Hello, I hope you find this link helpful.https://support.qq.com/embed/phone/415890/blog-archive

**Comment by Andy71m3** [comment]
Can I ask please the sync timer the app is it automatic if connected to the Web as it show 0
On my separate timer app it shows  say -0.2 but there is no setting on your app little confused but fantastic app so far

**Reply by Andy71m3** [reply]
Think it's been fixed thanks 😊

**Reply by UT8IIV** [reply]
Hello, I hope you find this link helpful. https://support.qq.com/embed/phone/415890/blog/751513

**Comment by sovegjarto** [comment]
I downloaded and import .adi file from qrz.com. There are closed QSO-s by qrz.com amateurs, but without LotW
Would be great to see these QSO-s (callsigns) in "closed" or "contacted" state too, similar to LotW. (Now only LotW QSO-s is managed as I see).

**Comment by R1BGW** [comment]
Доброго времени суток. Вопрос : Это приложение начиная с каких версий андроид  будет работать ?

**Reply by bg7yoz** [reply]
支持Android最小版本是6.0

**Comment by R1BGW** [comment]
Good time of day. It 's very strange that I don 't have android 10 support on my tablet .But it works on the phone 7 and 12 . I suspect that it 's possible on the tablet it 's about the firmware , although he sees the OTG cable .

**Comment by Andy71m3** [comment]
I am using a Bluetooth dongle in the back of the ft817 does the tx audio go out (get transmitted) via the dongle.
And what about the rx audio ? As when I turn the volume down it doesn't receive 73s all

**Comment by Andy71m3** [comment]
The more I use this program the better it becomes I still have a few learning curves.
For someone who operates portable qrp it's fab when you cannot hear anyone calling cq.

Can we use FT8cn for a contest there is a a space for the word pota sota could you put a serial number in there ? 73s all and keep up the fab program

**Reply by bg7yoz** [reply]
ft8协议要求 CQ modifier 是：000-999 , A-Z , AA-ZZ , AAA-ZZZ , AAAA-ZZZZ
例如： 
CQ POTA  callsign grid √
CQ 001 callsign grid √
CQ DX callsign grid √

CQ POTA001 callsign grid ×
CQ SOTA002 callsign grid ×

**Comment by K4DB** [comment]
Greetings! I've been using the app for a few months now and **really** like it! However, despite reading the "manual", etc., I cannot figure out how to get the app to initiate calling CQ initially? IF I respond to someone else's FT8 CQ, after the exchange is completed, it will then starting the CQ transmission process and work fine. I'm sure that it's something simple that I'm overlooking. I like to operate POTA, so the ability to start off calling CQ is important to me. Thanks & 73 - Daniel/K4DB

**Reply by Andy71m3** [reply]
I found it easier just to leave the app to run it all I and if the app doesn't get a contact after say 5 trys it goes into cq on its own its there in settings however never been successful cq-ing someone manually  hope this helps 73s

**Reply by bg7yoz** [reply]
一般情况下，在“Calling”页面，点击“CQ”按钮，就会进入CQ状态。

**Comment by LostBoy1981** [comment]
Hi,
I connected my smartphone (honor x5) with usdx+ by AUX,for both side radio and smartphone, but FT8CM not deciding.
So any advice plz?
Thanks
Ahmad / A61DQ

**Reply by bg7yoz** [reply]
hi, Ahmad,

ft8cn无法解码的原因，需要排除一下几个问题：

1.是不是没有接收到音频？方法：看看ft8cn的频谱图，如果能接收到音频，瀑布图会正确显示噪点和信号。如果不能接收到信号，瀑布图是黑色的。请确认ft8cn是不是没有录音的权限，或者aux的接头不正确。

2.是不是时间没有同步？方法：还是看ft8cn的频谱图，ft8的信号长度是12.64秒，信号应该在15秒周期之内。如果信号超出过多，是无法解码的。解决办法：在设置页面，修改时间偏移值，如果honor x5可以访问internet，也可以点击“sync”按钮同步时间。

3.是不是电台的操作模式不正确？ft8信号在各频段都是上边带（usb），如果是错误的操作模式如lsb，也是无法解码的。

好运！

bg7yoz

**Comment by LostBoy1981** [comment]
hi bg7yoz,thank you for replyi will try and update youand your FT*cn it's amazingA61DQ Ahmad




    On Wednesday, August 16, 2023 at 06:23:02 PM GMT+4, bg7yoz ***@***.***> wrote:  
 
 


hi, Ahmad,

ft8cn无法解码的原因，需要排除一下几个问题：

1.是不是没有接收到音频？方法：看看ft8cn的频谱图，如果能接收到音频，瀑布图会正确显示噪点和信号。如果不能接收到信号，瀑布图是黑色的。请确认ft8cn是不是没有录音的权限，或者aux的接头不正确。

2.是不是时间没有同步？方法：还是看ft8cn的频谱图，ft8的信号长度是12.64秒，信号应该在15秒周期之内。如果信号超出过多，是无法解码的。解决办法：在设置页面，修改时间偏移值，如果honor x5可以访问internet，也可以点击“sync”按钮同步时间。

3.是不是电台的操作模式不正确？ft8信号在各频段都是上边带（usb），如果是错误的操作模式如lsb，也是无法解码的。

好运！

bg7yoz

**Comment by BlueFalcon6** [comment]
I have tried running my IC 705 from wlan, turning the IC 705 into a access point after watching LY2H https://m.youtube.com/watch?v=UNH5mdsCJ38&t=942s but encounter an error "CIV stream: network transmission exception: sendto failed EINVAL".  Has anyone successfully done this with the 705 and Samsung S23U
![Screenshot_20240504_111012_Gallery](https://github.com/N0BOY/FT8CN/assets/168923633/d29840d2-eedb-4f6a-acea-c6e0aab6ec94)

**Comment by ua3unv** [comment]
Всем добрых времени суток! использую bluetooth-cat интерфейс, звук на передачу выдаётся в телефон. тоже самое происходит при подключении ВТ-гарнитуры, но при звонке гарнитура работает нормально. в приложении постоянно выскакивает предложение подключить ВТ-гарнитуру. как можно исправить? или приложение не доработано? но очень нравится. 73.

**Comment by ua3unv** [comment]
![S41115-21212621](https://github.com/user-attachments/assets/94401973-8b35-462f-b8f1-e75d26e76728)

**Comment by ra3trm** [comment]
hi！
不要忘了旧设备。
不是每个人都很富有。 我不是UA3REO。
请添加：YAESU, KENWOOD ,ICOM , BARRETT

**Comment by ea4jo** [comment]
Hola, nuevo en esta app. Una pregunta, alguien me podría decir por qué esta tachada esa línea?
Hago contactos pero la última línea, el 73,sale tachada y tengo que confirmar a mano. Gracias. 
![Screenshot_20251212_203116_com_bg7yoz_ft8cn_MainActivity](https://github.com/user-attachments/assets/46a87470-13c5-4a45-830f-dfc4e04e804d)
![Screenshot_20251212_202051_com_android_chrome_CustomTabActivity_edit_1087913991372075](https://github.com/user-attachments/assets/1a601a3f-bbc3-4db0-a914-51335052a26d)
